Modules/socketmodule.c

```diff
@@ -6220,8 +6241,9 @@ socket_socketpair(PyObject *self, PyObject *args)
     SOCKET_T sv[2];
     int family, type = SOCK_STREAM, proto = 0;
     PyObject *res = NULL;
+    socket_state *state = get_module_state(self);
 #ifdef SOCK_CLOEXEC
-    int *atomic_flag_works = &sock_cloexec_works;
+    int *atomic_flag_works = &(state->sock_cloexec_works);
```

Comment:
```suggestion
    int *atomic_flag_works = &state->sock_cloexec_works;
```

Nit: parens look weird to me.

Reply:
Fair enough